spinalcordtoolbox/scripts/sct_label_utils.py

```diff
@@ -108,22 +108,10 @@ def get_parser():
         action="store_true",
         help="Compute the center-of-mass for each label value."
     )
-    func_group.add_argument(
-        '-disc',
-        metavar=Metavar.file,
-        help=textwrap.dedent("""
-            Project disc labels (`-disc`) onto a spinal cord segmentation (`-i`) within the axial plane to create a labeled segmentation.
-
-            - Note: Unlike `sct_label_vertebrae -discfile`, this function does NOT involve cord straightening.
-            - Note: This method does NOT involve orthogonal projection onto the cord centerline. Details: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3395#issuecomment-1478435265
-
-            The disc labeling follows the convention: https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling/labeling-conventions.html
-        """),  # noqa: E501 (line too long)
-    )
     func_group.add_argument(
         '-project-centerline',
         metavar=Metavar.file,
-        help="Project disc labels onto the spinal cord centerline."
+        help="Project disc labels (`-project-centerline`) onto the spinal cord centerline using the spinal cord segmentation (`-i`)."
```

Comment:
What do you think about this wording? Basically, I wanted to say that we can also project other labels, e.g., rootlets center-of-mass.

```suggestion
        help="Project labels (e.g., disc labels) provided by this argument (`-project-centerline`) onto the spinal cord centerline using the spinal cord segmentation (provided by the `-i` argument)."
```

Reply:
Good idea !